src/vue-pages/register.vue

```diff
@@ -69,6 +69,22 @@
                 placeholder="Enter your password"
               />
             </div>
+            <div>
+              <label for="password" class="block text-sm font-medium text-gray-700">Captcha</label>
+              <div
+                v-if="hasCaptcha !== false"
+                class="cf-turnstile"
+                :data-sitekey="CLOUDFLARE_TURNSTILE_SITE_KEY"
+                data-size="flexible"
+              ></div>
+              <div v-if="hasCaptcha === false" class="flex items-start mt-4">
+                Please turn off your adblock!
```

Comment:
can you reference the translations object after adding this to en.yml?

Reply:
sovled in https://github.com/Cap-go/website/pull/250/commits/b20f3424322ff2973a3df789ddd50897b8ede4b4